.golangci.yml

```diff
@@ -40,6 +39,23 @@ linters:
     - unconvert
     - staticcheck
     - gosec
+    - tparallel
+    - whitespace
+    - revive
+    - godot
+    - errorlint
+    - gocritic
+    - errname
+    - govet
+    - predeclared
+    - nestif
+    - exhaustive
+    - tenv
+    - gofumpt
+    - forcetypeassert
+    - nilerr
+    - errcheck
+    # - promlinter this is a very nice linter, but it will most probably break things...
```

Comment:
This linter is really nice. It flags things like:
- help message not added
- used counter with no _totla suffix
- etc
It will most probably break things...